### ISSUE TITLE
Missing two sequences after initializing the vots2023 dataset

### ISSUE THREAD

**Issue by RobotVisionTsai** [votchallenge/toolkit#93]
Dear all,

I have initialized the vots2023 workspace, built a native tracker, and start running evaluation of the tracker.
However, when the evaluation process runs to 90%, an error occurs as follows:
File "C:\Users\chiyi\AppData\Roaming\Python\Python310\site-packages\vot\dataset\__init__.py", line 229, in __scan
    raise DatasetException("Empty sequence, no frames found.")
vot.dataset.DatasetException: Empty sequence, no frames found.

After checking the downloaded sequences, I find two sequences (train-7, transparent_fish) have no frames. 
I tried to reinitialize the workspace to download these two missing sequences (removing these two sequences from the workspace), but it didn't work.
Is there a command in vot-toolkit for users to download a specific sequence?

**Comment by lukacu** [comment]
Can you run inotialize with debug flag? which two sequences are problematic and are there always the same two?

**Comment by RobotVisionTsai** [comment]
Hi, Lukacu,

I used another computer to download the vots2023 dataset again, and this time it misses 10 different sequences. Fortunately, the missing sequences in two computers are different, so I finally completed the dataset by copying the missing sequences from the other computer.
Currently, I encounter an analysis problem after the evaluation process.
I will open another issue to describe the analysis problem.
Thanks.
